ci/Jenkinsfiles/build.groovy

```diff
@@ -396,8 +396,7 @@ pipeline {
           ----------------------------------------
           """
           sh """
-            git add .
-            git commit -m "Release ${VERSION}"
+            git commit -a -m "Release ${VERSION}"
```

Comment:
is this because since there is no new untracked file to be added any way at this point that you privilledge `-a` over `.` (and to have a one-liner ?)

Reply:
Yes for both reasons